unfoldingWord/translationCore
# White Screen Of Death

#### This pull request addresses:

This fixes #3025 and also fixes the `update-apps` npm command for windows systems.

#### How to test this pull request:

Create a windows and mac build via `npm make`. The builds should install/run without producing a white screen.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3047)
